static/docs/commands-reference/import.md

```diff
@@ -21,28 +12,278 @@ Supported schemes:
       out            Output
 ```
 
+## Description
+
+DVC supports `.dvc` files which refer to an external data file, see
+[External Dependencies](/doc/user-guide/external-dependencies). In such a DVC
+file, the `deps` section will list the remote file specification and the `outs`
+section will list the local file name in the workspace. It records enough data
+from the remote file to enable DVC to efficiently check the remote file to
+determine if the local file is out of date.
+
+The `dvc import` command helps the user create such an external data dependency.
+
+It generates a DVC file listing in the `deps` section the resource named in
+the `url` parameter, and in the `outs` section the file named in the `out`
+parameter.
+
+DVC supports several types of remote files:
+
+Type | Discussion | URL format
+-----|------------|------------
+`local` | Local path | `/path/to/local/file`
+`s3` | Amazon S3 | `s3://mybucket/data.csv`
+`gs` | Google Storage | `gs://mybucket/data.csv`
+`ssh` | SSH server | `ssh://user@example.com:/path/to/data.csv`
+`hdfs` | HDFS | `hdfs://user@example.com/path/to/data.csv`
+`http` | HTTP to file with _strong ETag_ | `https://example.com/path/to/data.csv`
```

Comment:
there is one more actually - `remote://`. See this ticket - https://github.com/iterative/dvc.org/issues/108. It would be great to add it here and propagate the explanation to the external dependencies section, and dvc run if necessary.

Reply:
The remote URL needs to be documented in the `dvc remote` command documentation.  It should then be enough to reference that documentation from here.

Reply:
I would still explain it briefly here - just give an example of the transformation.